src/store/modules/user/actions.ts

```diff
@@ -264,6 +264,32 @@ const actions: ActionTree<UserState, RootState> = {
       showToast(translate("Unable to fetch NetSuite RMA mapping List."));
     }
   },
+  async updateUserProfile({ commit, dispatch }, payload) {
```

Comment:
As this functions do not change the state we should move this to services instead of having actions.